src/Widgets/PartitionBar.vala

```diff
@@ -89,6 +91,10 @@ public class Installer.PartitionBar : Gtk.EventBox {
         return end - start;
     }
 
+    public uint64 get_sector_size () {
```

Comment:
This doesn't seem to be used anywhere, so could be removed?

Reply:
done!